proto/storage/compliance_operator_v2.proto

```diff
@@ -218,19 +219,22 @@ message ComplianceOperatorCondition {
   google.protobuf.Timestamp last_transition_time = 5;
 }
 
+// Next Tag: 5
+// ComplianceOperatorStatus represents an observation of an object's status.
+message ComplianceOperatorStatus {
+  string phase = 1;
+  string result = 2;
+  string error_message = 3;
+  repeated ComplianceOperatorCondition conditions = 4;
+}
+
 // Next Tag: 5
 // ComplianceOperatorSuiteV2 represents scan status
 message ComplianceOperatorSuiteV2 {
   string id = 1 [(gogoproto.moretags) = 'sql:"pk,id,type(uuid)"'];
   string name = 2 [(gogoproto.moretags) = 'search:"Compliance Suite Name,hidden"'];
   string cluster_id = 3 [(gogoproto.moretags) = 'search:"Cluster ID,hidden" sql:"fk(Cluster:id),no-fk-constraint,type(uuid)"'];
 
-  message Status {
-    string phase = 1;
-    string result = 2;
-    string error_message = 3;
-    repeated ComplianceOperatorCondition conditions = 4;
-  }
-
-  Status status = 4;
+  reserved 4;
```

Comment:
Because this is just moving the ComplianceOperatorStatus out and all the types and values are the same, we don't need to reserve